### ISSUE TITLE
Hide suggested videos tweaks not working. Interferes with „uYou Repeat“ tweak.

### ISSUE THREAD

**Issue by hi-im-karma** [arichornlover/uYouEnhanced#373]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Have you read the FAQ?

- [X] Yes, I did read the FAQ

### iOS/iPadOS version

17.3.1

### Device model

iPhone 14 Pro Max

### Sideload tool

MapleSign Pro

### Tweak version

uYouEnhanced 19.06.2-3.0.3

### Describe the bug

The latest update that I just installed has completely broken the uYou Repeat tweak in landscape mode. 
The „hide suggested videos at the end in full screen“ tweaks haven’t been working anyway. 

But now when uYou Repeat is enabled (I have it enabled often) the suggested videos pop up at the end in full screen, and the video repeats with suggested videos in the videoplayer while it’s playing. I recorded the bug:
https://imgur.com/gallery/JK0fAs5

![IMG_1810](https://github.com/arichornlover/uYouEnhanced/assets/73917917/60095e2a-5e16-47d1-a505-210e2bf8ae70)

![IMG_1809](https://github.com/arichornlover/uYouEnhanced/assets/73917917/92fb3063-1561-4221-bf5e-f818192aa1ec)

![IMG_1807](https://github.com/arichornlover/uYouEnhanced/assets/73917917/62e9f67f-74a4-495a-bc38-794234ef3154)

### Steps to reproduce the issue

1. Enable all „hide suggested videos at the end (in fullscreen) tweaks
2. Enable uYou Repeat button
3. Let video finish and see for yourself …

1. Turn off uYou Repeat button
2. Play video to the end.
3. Suggested videos pop up (despite having the tweaks enabled to not show them)

### Are you using the newest version of uYouPlus? If not, why?

✅ Yes, I'm using the latest version of uYouPlus right now

### Does the issue happen with the official YouTube from AppStore?

❌ No, I can't reproduce this issue in the official YouTube from AppStore

**Comment by hi-im-karma** [comment]
@arichornlover Can you please look into this issue when you have the time? ☝🏻

**Comment by hi-im-karma** [comment]
@bhackel could you please look into this some time? Its rly super annoying

**Comment by hi-im-karma** [comment]
> This is for the popup in landscape mode right? I think the only setting that should affect it is uYouEnhanced -> Hide Suggested Videos in Fullscreen, which must be broken. I can take a look this weekend when I have some time

Yes exactly. The pop up that shows suggested videos when a video ends. It even shows with the uyou repeat button enabled. I have described everything in detail and even posted a screen recording of the issue, everything above. Thank you so much if you get the opportunity to investigate this 🙏🏻

**Comment by KarmaBeast** [comment]
![image](https://github.com/arichornlover/uYouEnhanced/assets/73917917/3e0ae1cd-bd4b-4a89-a46e-df903c1fd27f)

Still an issue unfortunately.

@bhackel Could you please look into this some time?

**Comment by bhackel** [comment]
I'll add it to my list of things to do. I'm not even sure if the Hide Suggested Videos in Fullscreen option even works.

**Comment by KarmaBeast** [comment]
Thanks man 👌🏻 It's really annoying. "uYou repeat" is on the video ends and suddenly plays with those suggested videos popping up from the bottom 😐
